lib/SILOptimizer/Utils/InstOptUtils.cpp

```diff
@@ -2025,12 +2025,31 @@ SILValue swift::makeNewValueAvailable(SILValue value, SILBasicBlock *inBlock) {
   assert(value->getUses().empty() &&
          value.getOwnershipKind() == OwnershipKind::Owned);
 
+  return makeExistingValueAvailable(value, inBlock);
+}
+
+SILValue swift::makeExistingValueAvailable(SILValue value,
+                                           SILBasicBlock *inBlock) {
```

Comment:
This function is equivalent to `makeNewValueAvailable` (except the `assert(value->getUses().empty())`).
Why do we need a separate function? I looks like a simple rename to `makeExistingValueAvailable` of the old function would be sufficient.

Reply:
Yes, they are essentially the same except `makeExistingValueAvailable `  needs to consider uses. I added `makeExistingValueAvailable ` as a separate function to make this distinction. But it's probably making it more complex. I'll consolidate them.